joplin/templates/wagtailadmin/pages/index.html

```diff
@@ -0,0 +1,7 @@
+{% extends "wagtailadmin/pages/index.html" %}
+{% load wagtailadmin_tags staticfiles i18n %}
+
+{% block content %}
+{% include "wagtailadmin/pages/coa_header.html" %}
```

Comment:
probably would be a good place to include different headers depending on params (like 'are we searching?')